genai-perf/genai_perf/export_data/telemetry_data_exporter_util.py

```diff
@@ -0,0 +1,190 @@
+                )
+                for stat in STAT_COLUMN_KEYS:
+                    sub_table.add_column(stat, justify="right", style="green")
+
+                cls._construct_telemetry_stats_subtable(
+                    sub_table, metric_data, metric_name, STAT_COLUMN_KEYS
+                )
+                table.add_row(sub_table)
+
+            console.print(table)
+
+    @classmethod
+    def _construct_telemetry_stats_subtable(
+        cls,
+        table: Table,
+        metric_data: Dict[str, Dict[str, float]],
+        metric_name: str,
+        STAT_COLUMN_KEYS: List[str],
+    ) -> None:
+        gpu_indices = sorted([key for key in metric_data.keys() if key != "unit"])
+
+        for gpu_index in gpu_indices:
+            row = [f"{gpu_index}"]
+            for stat in STAT_COLUMN_KEYS:
+                value = metric_data.get(gpu_index, {}).get(stat, "N/A")
+                if isinstance(value, (float)):
+                    if metric_name == "gpu_utilization":
+                        value = str(int(round(value)))
+                    else:
+                        value = f"{value:,.2f}"
+                row.append(value)
+            table.add_row(*row)
+
+    @classmethod
+    def _write_dynamic_telemetry_stats(cls, telemetry_stats: Dict, csv_writer) -> None:
+        csv_writer.writerow([])
+        csv_writer.writerow(cls.TELEMETRY_DYNAMIC_METRICS_HEADER)
+
+        for metric_name, metric_data in telemetry_stats.items():
+            if metric_name in cls.TELEMETRY_CONSTANT_METRICS:
+                continue
+            metric_str = cls._format_metric_name(metric_name.replace("_", " "))
+            metric_str += f" ({metric_data['unit']})"
+
+            for key, gpu_data in metric_data.items():
+                if key == "unit":
+                    continue
+
+                row_values = [metric_str, key]
+
+                for stat in cls.TELEMETRY_DYNAMIC_METRICS_HEADER[2:]:
+                    value = gpu_data.get(stat, 0.0)
+                    row_values.append(f"{value:,.2f}")
+
+                csv_writer.writerow(row_values)
+
+    @classmethod
+    def _write_constant_telemetry_stats(cls, telemetry_stats: Dict, csv_writer) -> None:
+        csv_writer.writerow([])
+        csv_writer.writerow(cls.TELEMETRY_CONSTANT_METRICS_HEADER)
+
+        for metric_name, metric_data in telemetry_stats.items():
+            if metric_name not in cls.TELEMETRY_CONSTANT_METRICS:
+                continue
+
+            metric_str = cls._format_metric_name(metric_name.replace("_", " "))
+            metric_str += f" ({metric_data['unit']})"
+            for gpu in metric_data.keys():
+                if gpu == "unit":
+                    continue
+                value = metric_data[gpu]["avg"]
+                csv_writer.writerow([metric_str, gpu, f"{value:.2f}"])
+
+    @classmethod
+    def _format_metric_name(cls, metric_name: str) -> str:
+        abbreviations = ["gpu"]
```

Comment:
This should be a constant. You don't want magic values in your code.

Also, can all the constants be moved into their own file for maintainability?